notebooks/en/rag_with_hugging_face_gemma_mongodb.ipynb

```diff
@@ -0,0 +1,4011 @@
+{
```

Comment:
<p>It may be worth adding a small note why you don't do any chunking here. E.g. the examples in "fullplot" are already guaranteed to be short enough, or we don't care about truncating outliers. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>